### DISCUSSION TITLE
Add feature for Play Next Video (Up Next) aka SHIFT + N feature

### DISCUSSION THREAD

**Discussion by rcatank** [FreeTubeApp/FreeTube#4254] [General]
On Youtube native app, you can skip to the next video in the Up Next playlist (not to confuse with "Playlists" list) just by pressing SHIFT + N, or you can go back a video using SHIFT + P (previous). 

Can we get feature in Freetube? Thanks to all who have contributed coding.

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
There is already an open feature request covering this

https://github.com/FreeTubeApp/FreeTube/issues/2138#issuecomment-1073724876

But to answer your question yes this will be added eventually
